test/integration/ads_integration_test.cc

```diff
@@ -226,7 +226,9 @@ class AdsIntegrationTest : public HttpIntegrationTest, public Grpc::GrpcClientIn
       validation_context->add_verify_subject_alt_name("foo.lyft.com");
       if (clientType() == Grpc::ClientType::GoogleGrpc) {
         auto* google_grpc = grpc_service->mutable_google_grpc();
-        auto* ssl_creds = google_grpc->mutable_ssl_credentials();
+        auto* ssl_creds = google_grpc->add_credentials()
```

Comment:
Can you verify we have full coverage of all the new code paths with the tests in this PR?

Reply:
I'm having trouble running the coverage tests locally, however the tests should include both the default credentials (tests already existed) and the example credentials.